### PR TITLE
Add a checked mode banner to the module template default app

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

File: packages/flutter_tools/templates/module/common/lib/main.dart.tmpl [hunk 1]
```diff
@@ -6,7 +6,14 @@ import 'package:package_info/package_info.dart';
 
 Future<void> main() async {
   final PackageInfo packageInfo = await PackageInfo.fromPlatform();
-  runApp(Directionality(textDirection: TextDirection.ltr, child: Router(packageInfo)));
+  runApp(
+    Directionality(
+      textDirection: TextDirection.ltr,
+      child: new CheckedModeBanner(
```
Comment (mravn-google) [inline packages/flutter_tools/templates/module/common/lib/main.dart.tmpl]: Remove `new`

**Review by mravn-google — APPROVED** [review]
LGTM

File: packages/flutter_tools/templates/module/common/lib/main.dart.tmpl (hunk 1 above)
Comment (sigurdm) [reply packages/flutter_tools/templates/module/common/lib/main.dart.tmpl]: Done

**Comment by sigurdm** [comment]
This will not be needed as we change the default app in the module template

**Pull request closed without merging**